### PR TITLE
[drawer] Run button must redirect to that topology endpoint. #20446

### PR DESCRIPTION
**Pull request by amatosc**

### CODE REVIEW

File: sinfonier-web/app/controllers/Modules.java [hunk 1]
```diff
@@ -383,9 +383,13 @@ public static void recheck(String id, Integer versionCode) throws SinfonierExcep
             JsonObject res = client.uploadModule(module.getId(), moduleVersion.getId());
 
             if (res.get("code") != null && res.get("code").getAsInt() == Client.STATUS_SUCCESS) {
+              // Before to save it we should synchronized the version
+              moduleVersion = ModuleVersion.findById(moduleVersion.getId());
+
               // Recheck module version
```
Comment (demonodojo) [inline sinfonier-web/app/controllers/Modules.java:387]: Recheck must only update the field, not the full object

Comment (amatosc) [reply sinfonier-web/app/controllers/Modules.java:387]: recheck must have synchronize both objects before to save it, this do that.

Comment (fandrod) [reply sinfonier-web/app/controllers/Modules.java:387]: Con la solución actual, hay que volver a hacer una lectura adicional de la versión, para traer las posibles modificaciones realizadas sobre el registro por el BackEnd, para a continuación actualizar el registro completo.
Lo que propone Josemi es tan solo actualizar el campo del estado, lo que evita posibles problemas y tiene un mejor rendimiento.
With the current solution, you must do a additional reading of module version, to bring any changes made on the record by the BackEnd, then upgrade to the full record.
What Josemi proposes is only update the status field, avoiding potential problems and with a better performance.

Comment (amatosc) [reply sinfonier-web/app/controllers/Modules.java:387]: I have pushed a new commit where I only updated the status. please check it.

**Review by demonodojo — APPROVED** [review]

File: sinfonier-web/app/views/Secure/layout.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
   <link rel="stylesheet" type="text/css" media="screen" href="@{'/public/plugins/bootstrap/css/bootstrap.min.css'}"/>
   <link rel="stylesheet" type="text/css" media="screen" href="@{'/public/plugins/font-awesome/css/font-awesome.min.css'}"/>
   <link rel="stylesheet" type="text/css" media="screen" href="@{'/public/plugins/bootstrap-select/dist/css/bootstrap-select.min.css'}"/>
+  <link rel="stylesheet" type="text/css" media="screen" href="@{'/public/stylesheets/module.css'}"/>
```
Comment (fandrod) [inline sinfonier-web/app/views/Secure/layout.html:10]: Why we need this CSS here?

Comment (amatosc) [reply sinfonier-web/app/views/Secure/layout.html:10]: It's a css imported by darwin, as we replace this template by ours, we currently are missing some styles on the login screen.

**Review by fandrod — APPROVED** [review]

**Review by ivan11paths — APPROVED** [review]

**Pull request merged**